### PR TITLE
Mod/7925 mapbox update

### PR DESCRIPTION
**Pull request by nick-torres**
**High level description:**

update mapbox version

**JIRA Ticket:**
[DEV-7925](https://federal-spending-transparency.atlassian.net/browse/DEV-7925)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [ ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [ ] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

**Review by anjenkin — CHANGES_REQUESTED** [review]
Please do not merge this until we can determine if the upgrade will change our pricing for mapbox

**Comment by anjenkin** [comment]
@nick-torres We can close this PR for now.  We have to get approval to upgrade mapbox given the difference in the pricing model.  Also, the timing for when the map is loaded is changed with v2.  I don't think this should make a difference but we should probably test it.  Finally, mapbox v1 is still loading in the code (in index.ejs) so that should be removed to actually test mapbox2

The following is loaded in index.ejs

`<link href="https://api.mapbox.com/mapbox-gl-js/v1.13.0/mapbox-gl.css" rel="stylesheet" />
    <link rel="shortcut icon" href="img/favicon.ico">

    <script async type="text/javascript" id="_fed_an_ua_tag"
        src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=TREAS&subagency=FS"></script>
    
    <script src='https://api.mapbox.com/mapbox-gl-js/v1.13.0/mapbox-gl.js'></script>
    
    <script>
        window.ga = window.ga || function () { (ga.q = ga.q || []).push(arguments) }; ga.l = +new Date;
        ga('create', '<%= GA_TRACKING_ID %>', 'auto');
    </script>
    <script async src='https://www.google-analytics.com/analytics.js'></script>`

**Comment by nick-torres** [comment]
closing until we get final word on the pricing model

**Pull request closed without merging**
